pages/nfttest/mint/_iscnId.vue

```diff
@@ -74,28 +178,26 @@ export default class NFTTestMintPage extends Vue {
 
   async doAction() {
     /* eslint-disable no-fallthrough */
-    switch(this.state) {
+    switch (this.state) {
       case 'create':
-        await this.createNftClass();
-      case 'mint':
-        await this.sendNFT();
-        await this.postMintInfo();
-        await this.getMintInfo();
-        break;
+        this.isLoading = true
+        await this.createNftClass()
+        await this.sendNFT()
+        await this.postMintInfo()
+        await this.getMintInfo()
```

Comment:
if you catch the error in each function, you would have to check return value of each function, otherwise it will continue on and send wrong payload in the next step